cmd/http/main.go

```diff
@@ -21,6 +22,8 @@ import (
 	mflog "github.com/mainflux/mainflux/logger"
 	"github.com/mainflux/mainflux/pkg/messaging"
 	"github.com/mainflux/mainflux/pkg/messaging/brokers"
+	natstracing "github.com/mainflux/mainflux/pkg/messaging/nats/tracing"
```

Comment:
This should only be imported when using nats as the internal message broker hence import it on https://github.com/mainflux/mainflux/blob/master/pkg/messaging/brokers/brokers_nats.go